release-notes/README.md

```diff
@@ -4,6 +4,22 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## December 22, 2021
```

Comment:
let's also update the release notes under foundation-extensions/mobile-core/release-notes.md